### PR TITLE
[APM][DBM] Add dbm doc for PHP

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do?
Document the APM DBM link for PHP

### Motivation
The feature is on its way.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 1]
```diff
@@ -287,6 +290,23 @@ Enable the database monitoring propagation feature by setting the following envi
 
 {{% /tab %}}
 
+{{% tab "PHP" %}}
+
+<div class="alert alert-warning">
+This features requires automatic instrumentation to be enabled for your .NET service.
```
Comment (bwoebi) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:296]: Copy paste is fun, right?

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 2]
```diff
@@ -46,6 +46,9 @@ Data privacy
 |          | [Npgsql][16]                   | {{< X >}}   |             |
 |          | [MySql.Data][17]               |             | {{< X >}}   |
 |          | [MySqlConnector][18]           |             | {{< X >}}   |
+| **PHP**  [dd-trace-php][19] >= 0.86.0 ||             |             |
```
Comment (bwoebi) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:49]: 0.85.0 you mean? (just the restricting is going to be 0.86.0)

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 3]
```diff
@@ -287,6 +290,23 @@ Enable the database monitoring propagation feature by setting the following envi
 
 {{% /tab %}}
 
+{{% tab "PHP" %}}
+
+<div class="alert alert-warning">
+This features requires automatic instrumentation to be enabled for your .NET service.
+</div>
+
+Follow the [PHP tracing instructions][1] to install the automatic instrumentation package and enable tracing for your service.
```
Comment (bwoebi) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:299]: Is "automatic instrumentation package" a fancy term for "the tracer extension"?

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 4]
```diff
@@ -328,3 +348,6 @@ On the APM Service Page, view the direct downstream database dependencies of the
 [16]: https://www.nuget.org/packages/npgsql
 [17]: https://www.nuget.org/packages/MySql.Data
 [18]: https://www.nuget.org/packages/MySqlConnector
+[19]: https://github.com/DataDog/dd-trace-dotnet
```
Comment (bwoebi) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:351]: more C&P (at Pierres request: ;-))

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md (hunk 2 above)
Comment (pierotibou) [reply content/en/database_monitoring/guide/connect_dbm_and_apm.md:49]: I'd rather say `0.86.0` as it doesn't change anything and we'll be closer to the RFC with `0.86.0`

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 5]
```diff
@@ -46,6 +46,9 @@ Data privacy
 |          | [Npgsql][16]                   | {{< X >}}   |             |
 |          | [MySql.Data][17]               |             | {{< X >}}   |
 |          | [MySqlConnector][18]           |             | {{< X >}}   |
+| **PHP**  [dd-trace-php][19] >= 0.86.0 ||             |             |
+|          | [pdo][20]                       | {{< X >}}  | {{< X >}}   |
```
Comment (bwoebi) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:50]: Should probably also advertise mssql? Or is mssql not available yet from the DBM side itself?

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md (hunk 4 above)
Comment (pierotibou) [reply content/en/database_monitoring/guide/connect_dbm_and_apm.md]: What? We don't do .NET? :p

Comment (bwoebi) [reply content/en/database_monitoring/guide/connect_dbm_and_apm.md]: We absolutely do: https://php.net/Dotnet

**Review by alexandre-normand — APPROVED** [review]

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 6]
```diff
@@ -287,6 +290,23 @@ Enable the database monitoring propagation feature by setting the following envi
...
 
+{{% tab "PHP" %}}
+
+<div class="alert alert-warning">
+This features requires the tracer extension to be enabled for your PHP service.
+</div>
+
+Follow the [PHP tracing instructions][1] to install the automatic instrumentation package and enable tracing for your service.
+
+Ensure that you are using a supported client library (e.g. `PDO`).
```
Comment (estherk15) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:301]:
```suggestion
Ensure that you are using a supported client library. For example, `PDO`.
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 7]
```diff
@@ -356,6 +356,14 @@ Propagation styles to use when extracting tracing headers. If using multiple sty
   - [B3 single header][8]
   - Datadog
 
+`DD_DBM_PROPAGATION_MODE`
+: **INI**: `datadog.dbm_propagation_mode`<br>
+**Default**: `'disabled'`<br>
+: Enables linking between data sent from APM and the Database Monitoring product when set to `'service'` or `'full'`.<br>
```
Comment (estherk15) [inline content/en/tracing/trace_collection/library_config/php.md:362]: If this was intentional feel free to leave it. But if you want this to be consistent with formatting of other sections, you can remove the colon
image

```suggestion
Enables linking between data sent from APM and the Database Monitoring product when set to `'service'` or `'full'`.<br>
```

Comment (pierotibou) [reply content/en/tracing/trace_collection/library_config/php.md:362]: Thanks. Approved

**Pull request merged**